<!DOCTYPE html> <html lang="ja"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> 論文 | Kenji Fukushima lab </title> <meta name="author" content="Kenji Fukushima"> <meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar."> <meta name="keywords" content="plant, evolution, laboratory"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://kenji-fukushima-lab.github.io/publications/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?38c5c02344a0e46cb146102ac67bcdca"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/ja/"> Kenji Fukushima lab </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/ja/">概要 </a> </li> <li class="nav-item "> <a class="nav-link" href="/ja/blog/">ブログ </a> </li> <li class="nav-item active"> <a class="nav-link" href="/ja/publications/">論文 <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/ja/projects/">プロジェクト </a> </li> <li class="nav-item "> <a class="nav-link" href="/ja/repositories/">リポジトリ </a> </li> <li class="nav-item "> <a class="nav-link" href="/ja/cv/">履歴書 </a> </li> <li class="nav-item "> <a class="nav-link" href="/ja/teaching/">授業等 </a> </li> <li class="nav-item "> <a class="nav-link" href="/ja/people/">メンバー </a> </li> <li class="nav-item dropdown active"> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">その他 <span class="sr-only">(current)</span> </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item active" href="/ja/publications/">論文</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/ja/projects/">プロジェクト</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/ja/blog/">ブログ</a> </div> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">EN</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">論文</h1> <p class="post-description"> publications by categories in reversed chronological order. generated by jekyll-scholar. </p> </header> <article> <div class="publications"> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="fukushima2023decaploid" class="col-sm-8"> <div class="title">A decaploid pitcher plant genome reveals a novel role for recessive subgenomes</div> <div class="author"> Kenji Fukushima </div> <div class="periodical"> <em>Nature Plants</em>, Dec 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Genomic polyploidy is prevalent in the plant kingdom, giving rise to dominant and recessive subgenomes. We show that the recessive subgenomes of the pitcher plant Nepenthes gracilis had a crucial role in the acquisition of novel genes with species-specific function such as dioecy and carnivory.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Saul2023" class="col-sm-8"> <div class="title">Subgenome dominance shapes novel gene evolution in the decaploid pitcher plant Nepenthes gracilis</div> <div class="author"> Franziska Saul ,  Mathias Scharmann ,  Takanori Wakatake ,  Sitaram Rajaraman ,  André Marques ,  Matthias Freund ,  Gerhard Bringmann ,  Louisa Channon ,  Dirk Becker ,  Emily Carroll ,  Yee Wen Low ,  Charlotte Lindqvist ,  Kadeem J. Gilbert ,  Tanya Renner ,  Sachiko Masuda ,  Michaela Richter ,  Gerd Vogg ,  Ken Shirasu ,  Todd P. Michael ,  Rainer Hedrich ,  Victor A. Albert ,  and  Kenji Fukushima </div> <div class="periodical"> <em>Nature Plants</em>, Dec 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Subgenome dominance after whole-genome duplication generates distinction in gene number and expression at the level of chromosome sets, but it remains unclear how this process may be involved in evolutionary novelty. Here we generated a chromosome-scale genome assembly of the Asian pitcher plant Nepenthes gracilis to analyse how its novel traits (dioecy and carnivorous pitcher leaves) are linked to genomic evolution. We found a decaploid karyotype and a clear indication of subgenome dominance. A male-linked and pericentromerically located region on the putative sex chromosome was identified in a recessive subgenome and was found to harbour three transcription factors involved in flower and pollen development, including a likely neofunctionalized LEAFY duplicate. Transcriptomic and syntenic analyses of carnivory-related genes suggested that the paleopolyploidization events seeded genes that subsequently formed tandem clusters in recessive subgenomes with specific expression in the digestive zone of the pitcher, where specialized cells digest prey and absorb derived nutrients. A genome-scale analysis suggested that subgenome dominance likely contributed to evolutionary innovation by permitting recessive subgenomes to diversify functions of novel tissue-specific duplicates. Our results provide insight into how polyploidy can give rise to novel traits in divergent and successful high-ploidy lineages.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="MONTERO2023R497" class="col-sm-8"> <div class="title">Non-prey biotic interactions in carnivorous plants</div> <div class="author"> Héctor Montero ,  and  Kenji Fukushima </div> <div class="periodical"> <em>Current Biology</em>, Dec 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Carnivorous plants often spark broad interest due to their specialized adaptations for trapping and consuming animals. These notable organisms not only fix carbon through photosynthesis, but they also obtain essential nutrients such as nitrogen and phosphate from their captured prey. In typical angiosperms, interactions with animals are usually confined to such processes as pollination and herbivory, but another layer of complexity in these interactions is added for carnivorous plants. Here, we introduce carnivorous plants and their associated organisms — ranging from their prey to their symbionts — and highlight biotic interactions beyond carnivory to discuss how the ‘default’ interactions typical for flowering plants have changed in the case of the carnivorous plants (Figure 1).</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Fukushima2023NEE" class="col-sm-8"> <div class="title">Detecting macroevolutionary genotype–phenotype associations using error-corrected rates of protein convergence</div> <div class="author"> Kenji Fukushima ,  and  David D. Pollock </div> <div class="periodical"> <em>Nature Ecology &amp; Evolution</em>, Feb 2023 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="noauthororeditor" class="col-sm-8"> <div class="title">Transcriptomic heterochrony and completely cleistogamous flower development in the mycoheterotrophic orchid Gastrodia</div> <div class="author"> Kenji Suetsugu ,  Kenji Fukushima ,  Takashi Makino ,  Shuka Ikematsu ,  Tomoaki Sakamoto ,  and  Seisuke Kimura </div> <div class="periodical"> <em>New Phytologist</em>, Jan 2023 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="freund2022digestive" class="col-sm-8"> <div class="title">The digestive systems of carnivorous plants</div> <div class="author"> Matthias Freund ,  Dorothea Graus ,  Andreas Fleischmann ,  Kadeem J. Gilbert ,  Qianshi Lin ,  Tanya Renner ,  Christian Stigloher ,  Victor A. Albert ,  Rainer Hedrich ,  and  Kenji Fukushima </div> <div class="periodical"> <em>Plant Physiology</em>, Aug 2022 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Naito2022.03.28.486085" class="col-sm-8"> <div class="title">Genome sequence of 12 Vigna species as a knowledge base of stress tolerance and resistance</div> <div class="author"> Ken Naito ,  Takanori Wakatake ,  Tomoko F. Shibata ,  Kohtaro Iseki ,  Shuji Shigenobu ,  Yu Takahashi ,  Eri Ogiso-Tanaka ,  Chiaki Muto ,  Kuniko Teruya ,  Akino Shiroma ,  Makiko Shimoji ,  Kazuhito Satou ,  Takashi Hirano ,  Atsushi J. Nagano ,  Norihiko Tomooka ,  Mitsuyasu Hasebe ,  Kenji Fukushima ,  and  Hiroaki Sakai </div> <div class="periodical"> <em>bioRxiv</em>, Aug 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Harnessing plant genetic resources including wild plants enables exploitation of agronomically unfavorable lands to secure food in the future. The genus Vigna, family Fabaceae, consists of many species of such kind, as they are often adapted to harsh environments including marine beach, arid sandy soil, acidic soil, limestone karst and marshes. Here we report long-read assemblies of 12 Vigna species, achieving 95% or higher BUSCO scores. The comparative analyses discovered a new class of WUSCHEL-related homeobox (WOX) transcription factor superfamily that are incorporated into LTR retrotransposons and have dramatically amplified in some species of the genus Vigna. Except WOX transcription factors, however, gene contents are highly conserved among Vigna species with few copy number variations. On the other hand, transcriptome data provided some insights that transcriptional alterations played more important roles in evolution of stress tolerance in the genus Vigna. The whole genome sequences presented in this study will facilitate understanding genetic mechanisms of stress tolerance and application for developing new crops that are adapted to unfavorable environments.Competing Interest StatementThe authors have declared no competing interest.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="noauthororeditos" class="col-sm-8"> <div class="title">Gene expression evolution in pattern-triggered immunity within <i>Arabidopsis thaliana</i> and across Brassicaceae species</div> <div class="author"> Thomas M Winkelmüller ,  Frederickson Entila ,  Shajahan Anver ,  Anna Piasecka ,  Baoxing Song ,  Eik Dahms ,  Hitoshi Sakakibara ,  Xiangchao Gan ,  Karolina Kułak ,  Aneta Sawikowska ,  Paweł Krajewski ,  Miltos Tsiantis ,  Ruben Garrido-Oter ,  Kenji Fukushima ,  Paul Schulze-Lefert ,  Stefan Laurent ,  Paweł Bednarek ,  and  Kenichi Tsuda </div> <div class="periodical"> <em>The Plant Cell</em>, Aug 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Plants recognize surrounding microbes by sensing microbe-associated molecular patterns (MAMPs) to activate pattern-triggered immunity (PTI). Despite their significance for microbial control, the evolution of PTI responses remains largely uncharacterized. Here, by employing comparative transcriptomics of six Arabidopsis thaliana accessions and three additional Brassicaceae species to investigate PTI responses, we identified a set of genes that commonly respond to the MAMP flg22 and genes that exhibit species-specific expression signatures. Variation in flg22-triggered transcriptome responses across Brassicaceae species was incongruent with their phylogeny, while expression changes were strongly conserved within A. thaliana. We found the enrichment of WRKY transcription factor binding sites in the 5′-regulatory regions of conserved and species-specific responsive genes, linking the emergence of WRKY-binding sites with the evolution of gene expression patterns during PTI. Our findings advance our understanding of the evolution of the transcriptome during biotic stress.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="hedrich2021origin" class="col-sm-8"> <div class="title">On the origin of carnivory: Molecular physiology and evolution of plants on an animal diet</div> <div class="author"> Rainer Hedrich ,  and  Kenji Fukushima </div> <div class="periodical"> <em>Annual Review of Plant Biology</em>, Aug 2021 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="noauthororeditot" class="col-sm-8"> <div class="title">A discordance of seasonally covarying cues uncovers misregulated phenotypes in the heterophyllous pitcher plant Cephalotus follicularis</div> <div class="author"> Kenji Fukushima ,  Hideki Narukawa ,  Gergo Palfalvi ,  and  Mitsuyasu Hasebe </div> <div class="periodical"> <em>Proceedings of the Royal Society B</em>, Aug 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Organisms withstand normal ranges of environmental fluctuations by producing a set of phenotypes genetically programmed as a reaction norm; however, extreme conditions can expose a misregulation of phenotypes called a hidden reaction norm. Although an environment consists of multiple factors, how combinations of these factors influence a reaction norm is not well understood. To elucidate the combinatorial effects of environmental factors, we studied the leaf shape plasticity of the carnivorous pitcher plant Cephalotus follicularis. Clonally propagated plants were subjected to 12-week-long growth experiments in different conditions controlled by growth chambers. Here, we show that the dimorphic response of forming a photosynthetic flat leaf or an insect-trapping pitcher leaf is regulated by two covarying environmental cues: temperature and photoperiod. Even within the normal ranges of temperature and photoperiod, unusual combinations of the two induced the production of malformed leaves that were rarely observed under the environmentally typical combinations. We identified such cases in combinations of a summer temperature with a short-to-neutral day length, whose average frequency in the natural Cephalotus habitats corresponded to a once-in-a-lifetime event for this perennial species. Our results suggest that even if individual cues are within the range of natural fluctuations, a hidden reaction norm can be exposed under their discordant combinations. We anticipate that climate change may challenge organismal responses through not only extreme cues but also through uncommon combinations of benign cues.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="suda2020calcium" class="col-sm-8"> <div class="title">Calcium dynamics during trap closure visualized in transgenic Venus flytrap</div> <div class="author"> Hiraku Suda ,  Hiroaki Mano ,  Masatsugu Toyota ,  Kenji Fukushima ,  Tetsuro Mimura ,  Izuo Tsutsui ,  Rainer Hedrich ,  Yosuke Tamada ,  and  Mitsuyasu Hasebe </div> <div class="periodical"> <em>Nature Plants</em>, Aug 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The leaves of the carnivorous plant Venus flytrap, Dionaea muscipula (Dionaea) close rapidly to capture insect prey. The closure response usually requires two successive mechanical stimuli to sensory hairs on the leaf blade within approximately 30 s (refs. 1–4). An unknown biological system in Dionaea is thought to memorize the first stimulus and transduce the signal from the sensory hair to the leaf blade2. Here, we link signal memory to calcium dynamics using transgenic Dionaea expressing a Ca2+ sensor. Stimulation of a sensory hair caused an increase in cytosolic Ca2+ concentration ([Ca2+]cyt) starting in the sensory hair and spreading to the leaf blade. A second stimulus increased [Ca2+]cyt to an even higher level, meeting a threshold that is correlated to the leaf blade closure. Because [Ca2+]cyt gradually decreased after the first stimulus, the [Ca2+]cyt increase induced by the second stimulus was insufficient to meet the putative threshold for movement after about 30 s. The Ca2+ wave triggered by mechanical stimulation moved an order of magnitude faster than that induced by wounding in petioles of Arabidopsis thaliana5 and Dionaea. The capacity for rapid movement has evolved repeatedly in flowering plants. This study opens a path to investigate the role of Ca2+ in plant movement mechanisms and their evolution.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="fukushima2020amalgamated" class="col-sm-8"> <div class="title">Amalgamated cross-species transcriptomes reveal organ-specific propensity in gene expression evolution</div> <div class="author"> Kenji Fukushima ,  and  David D. Pollock </div> <div class="periodical"> <em>Nature Communications</em>, Aug 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The origins of multicellular physiology are tied to evolution of gene expression. Genes can shift expression as organisms evolve, but how ancestral expression influences altered descendant expression is not well understood. To examine this, we amalgamate 1,903 RNA-seq datasets from 182 research projects, including 6 organs in 21 vertebrate species. Quality control eliminates project-specific biases, and expression shifts are reconstructed using gene-family-wise phylogenetic Ornstein–Uhlenbeck models. Expression shifts following gene duplication result in more drastic changes in expression properties than shifts without gene duplication. The expression properties are tightly coupled with protein evolutionary rate, depending on whether and how gene duplication occurred. Fluxes in expression patterns among organs are nonrandom, forming modular connections that are reshaped by gene duplication. Thus, if expression shifts, ancestral expression in some organs induces a strong propensity for expression in particular organs in descendants. Regardless of whether the shifts are adaptive or not, this supports a major role for what might be termed preadaptive pathways of gene expression evolution.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="DREYER2020" class="col-sm-8"> <div class="title">How to Grow a Tree: Plant Voltage-Dependent Cation Channels in the Spotlight of Evolution</div> <div class="author"> Ingo Dreyer ,  Frances C. Sussmilch ,  Kenji Fukushima ,  Gonzalo Riadi ,  Dirk Becker ,  Jörg Schultz ,  and  Rainer Hedrich </div> <div class="periodical"> <em>Trends in Plant Science</em>, Aug 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Phylogenetic analysis can be a powerful tool for generating hypotheses regarding the evolution of physiological processes. Here, we provide an updated view of the evolution of the main cation channels in plant electrical signalling: the Shaker family of voltage-gated potassium channels and the two-pore cation (K+) channel (TPC1) family. Strikingly, the TPC1 family followed the same conservative evolutionary path as one particular subfamily of Shaker channels (Kout) and remained highly invariant after terrestrialisation, suggesting that electrical signalling was, and remains, key to survival on land. We note that phylogenetic analyses can have pitfalls, which may lead to erroneous conclusions. To avoid these in the future, we suggest guidelines for analyses of ion channel evolution in plants.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="PALFALVI20202312" class="col-sm-8"> <div class="title">Genomes of the Venus Flytrap and Close Relatives Unveil the Roots of Plant Carnivory</div> <div class="author"> Gergo Palfalvi ,  Thomas Hackl ,  Niklas Terhoeven ,  Tomoko F. Shibata ,  Tomoaki Nishiyama ,  Markus Ankenbrand ,  Dirk Becker ,  Frank Förster ,  Matthias Freund ,  Anda Iosip ,  Ines Kreuzer ,  Franziska Saul ,  Chiharu Kamida ,  Kenji Fukushima ,  Shuji Shigenobu ,  Yosuke Tamada ,  Lubomir Adamec ,  Yoshikazu Hoshi ,  Kunihiko Ueda ,  Traud Winkelmann ,  Jörg Fuchs ,  Ingo Schubert ,  Rainer Schwacke ,  Khaled Al-Rasheid ,  Jörg Schultz ,  Mitsuyasu Hasebe ,  and  Rainer Hedrich </div> <div class="periodical"> <em>Current Biology</em>, Aug 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Summary Most plants grow and develop by taking up nutrients from the soil while continuously under threat from foraging animals. Carnivorous plants have turned the tables by capturing and consuming nutrient-rich animal prey, enabling them to thrive in nutrient-poor soil. To better understand the evolution of botanical carnivory, we compared the draft genome of the Venus flytrap (Dionaea muscipula) with that of its aquatic sister, the waterwheel plant Aldrovanda vesiculosa, and the sundew Drosera spatulata. We identified an early whole-genome duplication in the family as source for carnivory-associated genes. Recruitment of genes to the trap from the root especially was a major mechanism in the evolution of carnivory, supported by family-specific duplications. Still, these genomes belong to the gene poorest land plants sequenced thus far, suggesting reduction of selective pressure on different processes, including non-carnivorous nutrient acquisition. Our results show how non-carnivorous plants evolved into the most skillful green hunters on the planet.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="zhang2019assembly" class="col-sm-8"> <div class="title">Assembly and annotation of a draft genome of the medicinal plant Polygonum cuspidatum</div> <div class="author"> Yonghong Zhang ,  Lanlan Zheng ,  Yan Zheng ,  Chao Zhou ,  Ping Huang ,  Xiao Xiao ,  Yongheng Zhao ,  Xincai Hao ,  Zhubing Hu ,  Qinhua Chen ,  Hongliang Li ,  Xuanbin Wang ,  Kenji Fukushima ,  Guodong Wang ,  and  Chen Li </div> <div class="periodical"> <em>Frontiers in Plant Science</em>, Aug 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Polygonum cuspidatum (Japanese knotweed, also known as Huzhang in Chinese), a plant that produces bioactive components such as stilbenes and quinones, has long been recognized as important in traditional Chinese herbal medicine. To better understand the biological features of this plant and to gain genetic insight into the biosynthesis of its natural products, we assembled a draft genome of P. cuspidatum using Illumina sequencing technology. The draft genome is ca. 2.56 Gb long, with 71.54% of the genome annotated as transposable elements. Integrated gene prediction suggested that the P. cuspidatum genome encodes 55,075 functional genes, including 6,776 gene families that are conserved in the five eudicot species examined and 2,386 that are unique to P. cuspidatum. Among the functional genes identified, 4,753 are predicted to encode transcription factors. We traced the gene duplication history of P. cuspidatum and determined that it has undergone two whole-genome duplication events about 65 and 6.6 million years ago. Roots are considered the primary medicinal tissue and transcriptome analysis identified 2,173 genes that were expressed at higher levels in roots compared to aboveground tissues. Detailed phylogenetic analysis demonstrated expansion of the gene family encoding stilbene synthase and chalcone synthase enzymes in the phenylpropanoid metabolic pathway, which is associated with the biosynthesis of resveratrol, a pharmacologically important stilbene. Analysis of the draft genome identified 7 abscisic acid and water deficit stress-induced protein-coding genes and 14 cysteine-rich transmembrane module genes predicted to be involved in stress responses. The draft de novo genome assembly produced in this study represents a valuable resource for the molecular characterization of medicinal compounds in P. cuspidatum, the improvement of this important medicinal plant, and the exploration of its abiotic stress resistance.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Fukushima2015" class="col-sm-8"> <div class="title">Oriented cell division shapes carnivorous pitcher leaves of Sarracenia purpurea</div> <div class="author"> Kenji Fukushima ,  Hironori Fujita ,  Takahiro Yamaguchi ,  Masayoshi Kawaguchi ,  Hirokazu Tsukaya ,  and  Mitsuyasu Hasebe </div> <div class="periodical"> <em>Nature Communications</em>, Dec 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The pitcher-shaped leaf of the carnivorous plant Sarracenia purpurea acts as a pitfall trap to capture small animals. Here, Fukushima et al. analyse pitcher leaf development and propose that this unusual shape evolved from ancestral planar leaves through changes in the orientation of cell division.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2014</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="doi:10.1002/dvg.22728" class="col-sm-8"> <div class="title">Adaxial–abaxial polarity: The developmental basis of leaf shape diversity</div> <div class="author"> Kenji Fukushima ,  and  Mitsuyasu Hasebe </div> <div class="periodical"> <em>genesis</em>, Dec 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Summary Leaves of flowering plants are diverse in shape. Part of this morphological diversity can be attributed to differences in spatiotemporal regulation of polarity in the upper (adaxial) and lower (abaxial) sides of developing leaves. In a leaf primordium, antagonistic interactions between polarity determinants specify the adaxial and abaxial domains in a mutually exclusive manner. The patterning of those domains is critical for leaf morphogenesis. In this review, we first summarize the gene networks regulating adaxial–abaxial polarity in conventional bifacial leaves and then discuss how patterning is modified in different leaf type categories. genesis 52:1–18, 2014. © 2013 The Authors. Genesis Published byWiley Periodicals, Inc.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2008</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Fukushima2008" class="col-sm-8"> <div class="title">Somatic chromosome differentiation in three species of the Byblis liniflora complex (Byblidaceae)</div> <div class="author"> Kenji Fukushima ,  Katsuya Nagano ,  and  Yoshikazu Hoshi </div> <div class="periodical"> <em>Chromosome Botany</em>, Dec 2008 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>We conducted chromosomal characterizations of three species in the Byblis liniflora complex using sequential fluorescent staining with chromomycin A3 (CMA) and 4’,6-diamidino-2-phenylindole (DAPI), and fluorescence in situ hybridization method. Byblis filifolia and B. rorida had the diploidal chromosome number of 2n=16, while B. liniflora had the tetraploidal chromosome number of 2n=32. All chromosomes of three species were median-centromeric. The primary constrictions, which contain centromeric regions, showed slightly CMA-positive stainability. Two CMA-positive DAPI-negative segments were observed on two chromosomes of the diploids B. rorida and B. filifolia, and the tetraploid B. liniflora. The 45S rDNA signals were detected on two chromosomes of the diploid and the tetraploid species, respectively. The positions of the 45S rDNA were corresponded to CMA-positive DAPI-negative segments. </p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Hoshi2008" class="col-sm-8"> <div class="title">Tandem repeat rDNA sequences derived from parents were stably maintained in hexaploids of Drosera spathulata complex (Droseraceae)</div> <div class="author"> Yoshikazu Hoshi ,  Junichi Shirakawa ,  Mitsuyasu Hasebe ,  Kenji Fukushima ,  and  Katsuhiko Kondo </div> <div class="periodical"> <em>Cytologia</em>, Sep 2008 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Drosera spathulata complex, which consists of diploid, tetraploid and hexaploid populations, is widely but disjunctively distributed from eastern part of Australia throughout the South East Asian countries, to Japan. Among these populations, high morphological changing has been found in this species. To investigate intraspecific DNA polymorphism, and to infer the polyploid origin, some populations and cultivars of the Drosera spathulata complex and close related species were investigated using molecular cluster analysis with nucleotide sequences of the large subunit of ribulose-1,5-bisphosphate carboxylase (rbcL), and the internal transcribed spacer (ITS) of 18S-26S nuclear rDNA sequences (ITS). The rbcL analysis to estimate the species as maternal inheritance revealed that the highest similarity to the hexaploid sequence was found in that of the tetraploid. In contrast, the nuclear rDNA analysis clarified that the hexaploid had two types of ITS sequences: One type of the sequence showed the highest sequence similarity to ITS in D. rotundifolia L. genome, while another type of sequence showed the highest sequence similarity to ITS in the tetraploid D. spathulata Labill. genome. These results suggested that the hexaploid genome could be derived from amphiploidizaion between ancestor species of D. rotundifolia as paternal genome, and the tetraploid D. spathulata as maternal genome. </p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2007</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Hoshi2007" class="col-sm-8"> <div class="title">A comparative study of karyotypes in two species of Byblis (Byblidaceae)</div> <div class="author"> Yoshikazu Hoshi ,  Kenji Fukushima ,  and  Katsuya Nagano </div> <div class="periodical"> <em>Chromosome Botany</em>, Sep 2007 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Byblis filifolia and B. liniflora had the diploidal and the tetraploidal chromosome numbers of 2n=16 and 32, respectively. In condensation behavior from prophase to metaphase, most chromosomes of both species had early condensing segments at the proximal regions. At prometaphase and early-metaphase, all chromosomes except for some small sized chromosomes had decondense segments at the distal regions in both arms. At mid-metaphase, well-spread chromosomes were quite rare to obtain, because all chromosomes at mid-metaphase stage were quite sticky nature, but not earlier stages of metaphase and prophase. The metaphase chromosomes in both species showed a gradual decrease in size from the largest to the smallest. In B. filifolia, total chromosome length at early-metaphase was 40.5 μm, while total chromosome length at mid-metaphase was 23.3 μm. In contrast, B. liniflora showed total chromosome length of 106.1 μm at early-metaphase, and 54.0 μm at mid-metaphase. To compare to the same stage of metaphase, the total length in B. liniflora was nearly twice as long as that of B. filifolia. Two sat-chromosomes were observed in both species. </p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Kenji Fukushima. Desenvolvido em <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> com o tema <a href="https://github.com/george-gca/multi-language-al-folio" rel="external nofollow noopener" target="_blank">multi-language-al-folio</a>. Hospedado por <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Fotos do <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. Last updated: January 17, 2024. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?2930004b8d7fcd0a8e00fdcfc8fc9f24"></script> <script defer src="/assets/js/common.js?fe2c0c904506204d06f5dee314bd8b77"></script> <script defer src="/assets/js/copy_code.js?7254ae07fe9cc5f3a10843e1c0817c9c" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>