<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta name="google-site-verification" content="B9LXqkorsqpa6QAJroGuBvAAEF184IgR5hRZO95-xH4"> <meta http-equiv="Permissions-Policy" content="interest-cohort=()"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Approaches | Fukushima lab </title> <meta name="author" content="Kenji Fukushima"> <meta name="description" content=""> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%8C%B1%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://kenji-fukushima-lab.github.io/projects/2_project/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?bf50d6d9dd867d3e0f3b0add94449649"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> Fukushima lab </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">people </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/research/">research </a> </li> <li class="nav-item "> <a class="nav-link" href="/resources/">resources </a> </li> <li class="nav-item "> <a class="nav-link" href="/outreach/">outreach </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog </a> </li> <li class="nav-item "> <a class="nav-link" href="/join/">join us </a> </li> <li class="nav-item "> <a class="nav-link" href="/access/">access </a> </li> <li class="nav-item active"> <a class="nav-link" href="/ja/projects/2_project/">JA</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">Approaches</h1> <p class="post-description"> </p> </header> <article> <p>In our laboratory, we specialize in solving complex problems by integrating methods from a diverse range of research disciplines. This multidisciplinary approach is tailored for each project we undertake. Specifically, we frequently combine methodologies from the following areas:</p> <h4 id="evolutionary-biology">Evolutionary Biology</h4> <p>Just as we, as individuals, bear subtle differences from our parents, living organisms subtly alter their forms with each successive generation. Viewing the passage of time on a grand scale reveals a fascinating narrative: primitive bacteria evolved over approximately 4 billion years into diverse forms, including humans, frogs, and carnivorous plants. This perspective, while accurate, only skims the surface of the complex mechanisms and pivotal events that have driven evolution at crucial junctures. Our commitment lies in delving deep into these mysteries to uncover the intricacies of evolutionary history.</p> <div align="center"> <img src="/assets/img/projects/heterotrophs_03.jpg" alt="Carnivorous plant evolution" width="600"> </div> <div style="margin-top: 30px;"></div> <h4 id="plant-molecular-biology">Plant Molecular Biology</h4> <p>Unlike animals, which can relocate to more favorable environments when faced with extreme temperatures or scarcity of food, plants are rooted to their spot and must adapt their physiology to survive. This resilience of plants under various environmental pressures is a central focus of our research.</p> <p>Molecular biology, with its emphasis on nucleic acids and proteins, underpins much of modern biology and is a critical tool in our laboratory work. In plant research, experiments must be tailored to address the unique characteristics of plant cells. This includes strategies for removing secondary metabolites during the purification of nucleic acids, as well as the use of Agrobacterium for the genetic modifications. We are actively advancing genetic modification techniques for carnivorous plants, expanding the capabilities within the field of plant molecular biology.</p> <div align="center"> <img src="/assets/img/projects/cephalotus_molecular_biology.jpg" alt="Plant Molecular Biology" width="600"> </div> <div style="margin-top: 30px;"></div> <h4 id="evo-devo">Evo-Devo</h4> <p>The complex structure of carnivorous trapping leaves presents a fascinating puzzle regarding their evolution. The complexity of these specialized organs doesn’t appear all at once but is progressively layered on during the development process. Intriguingly, at the onset of development, there is little to distinguish between the incipient forms of normal and trapping leaves, both of which start simple and remarkably similar. By closely observing the divergence in development between these two types of leaves over time, we can deduce the critical evolutionary milestones that led to the unique morphology of trapping leaves.</p> <div align="center"> <img src="/assets/img/projects/leaf_development.jpg" alt="Leaf Evo-Devo" width="300"> </div> <div style="margin-top: 30px;"></div> <h4 id="plant-physiology--electrophysiology">Plant physiology &amp; electrophysiology</h4> <p>Characteristics beyond mere appearance are of paramount importance. For example, to understand the digestive capabilities and absorption mechanisms of carnivorous plants, one must explore the knowledge of nutrient transport, metabolism, and plant hormones, which are integral to plant physiology.</p> <p>‘Electricity’ also plays a pivotal role in this context. While the concept of bioelectricity might immediately bring to mind muscles and nerves, plants, too, leverage electrical signals. At the heart of bioelectricity is what’s known as “membrane potential”, a term that describes the voltage difference arising from the disparity in ion composition on either side of a biological membrane. Electrophysiological methods, which are employed to detect these membrane potentials, are instrumental not only in analyzing the motility of plants like the snapping movement of Venus flytrap but also in understanding functions where ion transport is key, such as the digestive and absorption capabilities found broadly among carnivorous plants. While Kenji may not have practical experience with these techniques, there is a keen interest in enhancing the lab capabilities in this area, pending the recruitment of team members with the necessary expertise. In the meantime, electrophysiological analyses are being undertaken in collaboration with external partners.</p> <div align="center"> <img src="/assets/img/projects/cephalotus_gland.jpg" alt="Stoma-to-stomach evolution" width="450"> </div> <div style="margin-top: 30px;"></div> <h4 id="biochemistry--protein-engineering">Biochemistry &amp; Protein engineering</h4> <p>The capacity to synthesize and modify proteins in vitro opens up a significant pathway for obtaining detailed insights into biological evolution. Through biochemistry and protein engineering have the opportunity to simulate and scrutinize evolutionary pathways, illuminating the processes by which proteins have adapted and diversified to fulfill new roles and acquire novel characteristics.</p> <div align="center"> <img src="/assets/img/projects/protein_structure.jpg" alt="Protein structure" width="600"> </div> <div style="margin-top: 30px;"></div> <h4 id="synthetic-biology">Synthetic biology</h4> <p>The “build to understand” methodology holds substantial value in scientific research, offering a tangible means to validate theoretical knowledge through practical application. For example, in the realm of botany, particularly with carnivorous plants, our comprehension might be deepened not just by study but by attempting to replicate their characteristics in non-carnivorous plants. While completely transforming <em>Arabidopsis thaliana</em> into a <em>Nepenthes</em>-like full carnivore remains beyond our current capabilities, it is feasible to transfer specific traits with a few genetic modifications.</p> <p>A more attainable goal may involve mutating Arabidopsis proteins to mirror those found in carnivorous plants, effectively imbuing them with similar properties. Beginning with ancestral proteins, this approach can be viewed as an experimental replay of protein evolution.</p> <div align="center"> <img src="/assets/img/projects/carnivorization.jpg" alt="Carnivorization" width="300"> </div> <div style="margin-top: 30px;"></div> <h4 id="genomics">Genomics</h4> <p>The immense wealth of data contained within genomes provides key insights into the origins and traits of living organisms. Decoding the complete genomic sequences, predicting gene structures, and inferring their functions represent foundational steps in unlocking biological discoveries from genomic data. The advent of long-read sequencing technologies, such as those offered by PacBio and Oxford Nanopore, in conjunction with the Hi-C chromosome conformation capture methods, enables us to achieve genome assemblies at a chromosomal level.</p> <div align="center"> <img src="/assets/img/projects/dna-sequencing-concept-illustration.jpg" alt="Genomics" width="300"> </div> <div style="margin-top: 30px;"></div> <h4 id="bioinformatics">Bioinformatics</h4> <p>In the exploration of complex biological phenomena, bioinformatics emerges as a profoundly influential tool. The proficiency in utilizing existing tools for omics analysis and comparative genomics already yields substantial insights. However, our laboratory is committed not just to mastering these tools but also to forging new methodologies.</p> <div align="center"> <img src="/assets/img/projects/csubst_code.png" alt="Programming" width="600"> </div> <div style="margin-top: 30px;"></div> <h4 id="phylogenetics">Phylogenetics</h4> <p>Phylogenetics stands as a foundational pillar within bioinformatics, providing essential methodologies for engaging in comparative genomics. Beyond merely utilizing molecular phylogeny to delineate relationships among organisms, our laboratory frequently employs phylogenetic comparative methods. These methods enable us to deduce the evolutionary pathways of gene expression levels and other various traits within the context of phylogenetic trees.</p> <div align="center"> <img src="/assets/img/projects/ou.jpg" alt="Ornstein-Uhlenbeck models" width="300"> </div> <div style="margin-top: 30px;"></div> <h4 id="statistics">Statistics</h4> <p>In modern biology, mastering statistical principles is essential. Our daily research activities provide a rich environment for developing a wide range of statistical skills.</p> <div align="center"> <img src="/assets/img/projects/gwas.jpg" alt="Genome-wide association study" width="600"> </div> <div style="margin-top: 30px;"></div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Kenji Fukushima. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/george-gca/multi-language-al-folio" rel="external nofollow noopener" target="_blank">multi-language-al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Last updated: February 03, 2024. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?2930004b8d7fcd0a8e00fdcfc8fc9f24"></script> <script defer src="/assets/js/common.js?4a129fbf39254905f505c7246e641eaf"></script> <script defer src="/assets/js/copy_code.js?7254ae07fe9cc5f3a10843e1c0817c9c" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script async src="https://www.googletagmanager.com/gtag/js?id=G-19M9F0JMSS"></script> <script>function gtag(){window.dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","G-19M9F0JMSS");</script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>